frontend/src/components/Navbar.js

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from 'react';
+import {
+  Collapse,
+  Navbar,
+  NavbarToggler,
+  NavbarBrand,
+  Nav,
+  NavItem,
+  NavLink,
+} from 'reactstrap';
+
+const MyNavBar = (props) => {
```

Comment:
Great work! Can you change this from a constant to a functional component. [Here](https://imgur.com/CO0B4U6) is an example of how to do so.

Reply:
I'll take a look at it tonight after class!